pkg/cmd/logout.go

```diff
@@ -0,0 +1,40 @@
+package cmd
+
+import (
+	"context"
+
+	"github.com/observatorium/obsctl/pkg/config"
+	"github.com/spf13/cobra"
+)
+
+func NewLogoutCmd(ctx context.Context) *cobra.Command {
+	var tenantName, apiName string
+	cmd := &cobra.Command{
+		Use:   "logout",
+		Short: "Logout a tenant. Will remove locally saved details.",
+		Long:  "Logout a tenant. Will remove locally saved details.",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			conf, err := config.Read(logger)
+			if err != nil {
+				return err
+			}
+
+			return conf.RemoveTenant(logger, config.TenantName(tenantName), config.APIName(apiName))
+		},
+	}
+
+	cmd.Flags().StringVar(&tenantName, "tenant", "", "The name of the tenant to logout.")
+	cmd.Flags().StringVar(&apiName, "api", "", "The name of the API the tenant is associated with.")
+
+	err := cmd.MarkFlagRequired("tenant")
+	if err != nil {
+		panic(err)
+	}
+
+	err = cmd.MarkFlagRequired("api")
```

Comment:
Similarly to previous suggestion, we could skip API as required if we detect only single API is defined.

Reply:
Marked this flag as optional!